Formula/csvq.rb

```diff
@@ -0,0 +1,31 @@
+class Csvq < Formula
+  desc "SQL-like query language for csv"
+  homepage "https://mithrandie.github.io/csvq"
+  url "https://github.com/mithrandie/csvq/archive/v1.11.1.tar.gz"
+  sha256 "e6423098bc9ef4bd54d482331a3a8a88dc90b15ed2978ba7860fffba46765c68"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "csvq"
+  end
+
+  test do
+    system "#{bin}/csvq", "--version"
+
+    (testpath/"test.csv").write <<~EOS
+      a,b,c
+      1,2,3
+    EOS
+    expected = <<~EOS
+      +---+---+---+
+      | a | b | c |
+      +---+---+---+
+      | 1 | 2 | 3 |
+      +---+---+---+
+    EOS
+    result = shell_output("#{bin}/csvq 'SELECT * FROM `test.csv`'")
```

Comment:
It might be more robust to pick out one or two individual elements rather than print the entire table. That way it will be less likely to break if they make cosmetic changes to the layout.

Reply:
> It might be more robust to pick out one or two individual elements rather than print the entire table. That way it will be less likely to break if they make cosmetic changes to the layout.

I see. I fixit.